src/lib/Preview.js

```diff
@@ -1015,6 +1015,11 @@ class Preview extends EventEmitter {
                 this.enableViewers(viewerName);
             }
         });
+
+        // Features
+        // This makes features available everywhere that features is passed in, which includes
+        // all of the viewers for different files
+        this.options.features = options.features;
```

Comment:
Can we move this before the enable/disable viewer logic? Should it have a default value, such as an empty object?

Reply:
Good point, will do